functions/src/wort/handler.clj

```diff
@@ -8,6 +8,8 @@
 (defroutes app-routes
   (GET "/" []
     (resp/content-type (resp/resource-response "index.html" {:root "public"}) "text/html"))
+  (GET "/sound" []
```

Comment:
Ah. I think there's a teensy bit of confusion here. Don't worry about this route for now.

What I was suggesting in the issue is that you have a route that calls the phrase building clojure code. That's probably enough work/headache to warrant a separate PR.